### PR TITLE
Create custom instrument to visualize network traffic #13

### PR DESCRIPTION
**Pull request by Regno**
Instrument can receive data from signpost and show it as a list. Without any graphic output.

Task for make graphic output is #28.

### CODE REVIEW

File: CustomInstruments/MockLife/MockLife.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -21,6 +22,7 @@
 		840D6EAA22200C85004BB78C = {
 			isa = PBXGroup;
 			children = (
+				558F37FC23BCD00E00ED40D3 /* README.md */,
 				840D6EB322200C86004BB78C /* MockLife */,
```
Comment (victoriaqb) [inline CustomInstruments/MockLife/MockLife.xcodeproj/project.pbxproj]: MockIsAlive?

File: CustomInstruments/SampleProject/SampleProject.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -32,20 +34,21 @@
 /* Begin PBXCopyFilesBuildPhase section */
 		550BC7DA23B91317001C943B /* Embed Frameworks */ = {
 			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
+			buildActionMask = 12;
```
Comment (victoriaqb) [inline CustomInstruments/SampleProject/SampleProject.xcodeproj/project.pbxproj]: что бы это могло быть

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # Custom instruments
 
-В папке CustomInstruments находится пример создания своего инструмента для Xcode с использоваем языка CLIPS
+В папке CustomInstruments находятся примеры создания инструментов для Xcode с использоваем языка CLIPS:
 
-[Readme](CustomInstruments/README.md)
+[Mock Is A Live](CustomInstruments/MockLife/README.md)
```
Comment (victoriaqb) [inline README.md]: Alive

File: CustomInstruments/SampleProject/SampleProject.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Regno) [reply CustomInstruments/SampleProject/SampleProject.xcodeproj/project.pbxproj]: 🤷🏼‍♂️

**Pull request merged**